pkg/runtime/docker/client.go

```diff
@@ -105,8 +106,20 @@ func (dc *dockerClient) AttachContainer(container string) (err error) {
 func (dc *dockerClient) RunContainer(image string, config *runtime.ContainerConfig, name string) (string, error) {
 	portBindings := make(nat.PortMap)
 	for _, portMapping := range config.PortBindings {
-		portBindings[nat.Port(fmt.Sprintf(portFormat, portMapping.VMPort))] = []nat.PortBinding{
+		var hostIP string
+		if portMapping.BindAddress != nil {
+			hostIP = portMapping.BindAddress.String()
+		}
+
+		protocol := portMapping.Protocol
+		if len(protocol) == 0 {
+			// Docker uses TCP by default
+			protocol = meta.ProtocolTCP
```

Comment:
create a `const DefaultProtocol = ProtocolTCP` in meta?

Reply:
The default protocol is runtime implementation specific, for Docker it just happens to be TCP, so I wouldn't make it generic.

Reply:
ok, sure